client/src/components/common/Table.tsx

```diff
@@ -29,6 +29,23 @@ export const Table = styled.table`
       padding: 0.25rem 0.5rem;
     }
   }
+  @media screen and (max-width: 576px) {
```

Comment:
Should this be a screen breakpoint?

Reply:
Yeah I put it that way cause it's also a number instead of a bp above so I wondered if there was a reason to not import more stuff just for that and kept the same logic.